charts/telemetry-server/scripts/pre-init-checks.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+set -eu
+set -o pipefail
+
+echo "Checking PostgreSQL connectivity"
+
+check_table() {
```

Comment:
It is a good habit to declare local variables as being local, especially if the same names are being used in the calling context, or globally...

```suggestion
check_table() {
    local db table result
```

Reply:
Resolved in latest commit